tasks/terraform_destroy.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require_relative '../lib/cli_helper.rb'
+require 'json'
+require 'open3'
+
+class TerraformDestroy < TaskHelper
+  def destroy(opts)
+    dir = File.expand_path(opts[:dir], opts[:_boltdir]) if opts[:dir]
```

Comment:
I wonder if we should abstract the CLI flag building into the helper. [According to the docs](https://www.terraform.io/docs/commands/destroy.html) the only difference is that `apply` can take a terraform plan file as an argument, and even then it seems like `destroy` throws a sane error if you try to run it and pass in a plan file. Having all this logic be copied in 2 files seems like it just runs the risk of the tasks diverging in weird ways.

Reply:
Yeah, i thought of doing that too. This moved the needle. Refactored.